plugins/clnrest/README.md

```diff
@@ -0,0 +1,47 @@
+# CLNRest
+
+CLNRest is a lightweight Python-based core lightning plugin that transforms RPC calls into a REST service. By generating REST API endpoints, it enables the execution of Core Lightning's RPC methods behind the scenes and provides responses in JSON format.
+
+## Installation
+
+Install required packages with `pip install json5 flask flask_restx gunicorn pyln-client` or `pip install -r requirements.txt`.
```

Comment:
There does not seem to be a `requirements.txt` file.

Reply:
Pushed the `requirements.txt` but please note that I am still working on some more features like CORS, CSRF and CSP. This might change the requirements a bit in future.